static/js/project/services/project.service.js

```diff
@@ -56,7 +58,7 @@
           modules: [
```

Comment:
wrt  ```name: 'Prototype Task'``` 
At present prototype task and future milestone are generic modules. Let's add a boolean flag to differentiate between both. This will help  related topic such as use of prototype task for ranking evaluation and so on. Backend model can be updated to hold this field.